src/test/java/com/exasol/adapter/dynamodb/DynamodbAdapterCreateAndDropTestLocalIT.java

```diff
@@ -0,0 +1,77 @@
+package com.exasol.adapter.dynamodb;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import java.sql.SQLException;
+import java.util.concurrent.TimeoutException;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import com.exasol.ExaMetadata;
+import com.exasol.adapter.request.CreateVirtualSchemaRequest;
+import com.exasol.adapter.request.DropVirtualSchemaRequest;
+import com.exasol.bucketfs.BucketAccessException;
+import com.exasol.containers.ExasolContainer;
+
+/**
+ * Tests the {@link DynamodbAdapter#createVirtualSchema(ExaMetadata, CreateVirtualSchemaRequest)} and
+ * {@link DynamodbAdapter#dropVirtualSchema(ExaMetadata, DropVirtualSchemaRequest)} using a local docker version of
+ * DynamoDB and a local docker version of exasol.
+ **/
+@Tag("integration")
+@Testcontainers
+public class DynamodbAdapterCreateAndDropTestLocalIT {
```

Comment:
Maybe this test can be merged with some other integration test because it doesn't make a lot of testing but the creation of containers is quite expensive. It's a luxury to start two containers for one test method :)

Reply:
in the other integration test classes I don't want create the schema in each test case. Instead there it is a precondition. If an exception is thrown in the beforAll there is however no good error reporting.  Therefore I moved this one out.

Reply:
You can create schemas inside these two test methods and use another schema for other tests.

Reply:
This does not work, because I need different afterAll code.